### ISSUE TITLE
Optimization of Gazebo execution inside the RoboticsAcademy Docker Image (RADI) for Windows (+Nvidia graphics card)

### ISSUE THREAD

**Issue by jmplaza** [JdeRobot/RoboticsAcademy#902]
It is now feasible: [https://devblogs.microsoft.com/commandline/the-initial-preview-of-gui-app-support-is-now-available-for-the-windows-subsystem-for-linux-2/#bonus-leverage-wsls-gpu-access-to-run-linux-applications-with-3d-acceleration](https://devblogs.microsoft.com/commandline/the-initial-preview-of-gui-app-support-is-now-available-for-the-windows-subsystem-for-linux-2/#bonus-leverage-wsls-gpu-access-to-run-linux-applications-with-3d-acceleration)

**Comment by jmplaza** [comment]
Can we safely close this issue @dpascualhe , updating the instructions inside the [Robotics Academy UserGuide ](https://jderobot.github.io/RoboticsAcademy/user_guide/#windows) for Windows users to "Enable GPU Acceleration" ?

**Comment by dpascualhe** [comment]
Bringing this issue back to life now that GPU acceleration for Linux is in a much more stable position (https://github.com/JdeRobot/RoboticsInfrastructure/issues/441)

**Comment by jmplaza** [comment]
@dpascualhe do you think this issue is solved? Maybe @CDAM2020 @miniesda you can test it

**Comment by dpascualhe** [comment]
@jmplaza Yes I think it is solved. There has been several successful attempts of GPU acceleration in Windows. If someone else tests it and finds any problem a new issue (more specific) should be opened.
